lib/stream_data.ex

```diff
@@ -2094,6 +2094,85 @@ defmodule StreamData do
     end)
   end
 
+  @doc """
+  Generates arbitrary (past and future) dates.
+
+  Generated values shrink towards `Date.utc_today/0`.
+  """
+  @spec date() :: t(Date.t())
+  def date do
+    date([])
+  end
```

Comment:
Can we remove this in favor of `[]` options in the other clause? That way for example we can say `:origin` *defaults* to `Date.utc_today/0`.